### ISSUE TITLE
Anet Feature issue

### ISSUE THREAD

**Issue by Lonicer** [jinhyunj/EaTR#1]
Your work is very good, I want to know more about it, but your paper doesn’t seem to show how Anet’s CLIP text feature is obtained, and Anet’s visual feature is: sub_activitynet_v1-3.c3d.hdf5?

**Comment by wjun0830** [comment]
Can you reproduce the results for anet and Charades? @Lonicer

**Comment by Lonicer** [comment]
> Can you reproduce the results for anet and Charades? @Lonicer

He did not give the anet data reading file. I only conducted experiments on the Charades data set, and it felt much lower, as follows:
{
     "brief": {
         "MR-full-R1@0.5": 49.87,
         "MR-full-R1@0.7": 26.83,
         "MR-full-R5@0.5": 65.46,
         "MR-full-R5@0.7": 34.35,
         "MR-full-mAP": 28.54,
         "MR-full-mAP@0.5": 57.39,
         "MR-full-mAP@0.75": 24.63,
         "MR-middle-mAP": 31.58,
         "MR-short-mAP": 27.73
     },

**Comment by wjun0830** [comment]
Can you also share the opt.json file for that result?

Here are my reproduced results for ANet which are much below their reported ones.
For visual features, we used sub_activitynet_v1-3.c3d.hdf5.
        "MR-full-R1@0.5": 39.28,
        "MR-full-R1@0.7": 22.05,
        "MR-full-R5@0.5": 66.19,
        "MR-full-R5@0.7": 40.32,
        "MR-full-mAP": 27.64,
        "MR-full-mAP@0.5": 50.95,
        "MR-full-mAP@0.75": 24.84,
        "MR-middle-mAP": 24.05,
        "MR-short-mAP": 15.1

[opt.zip](https://github.com/jinhyunj/EaTR/files/12544687/opt.zip)

**Comment by Lonicer** [comment]
Are you using start_end_dataset.py provided by EaTR for data reading operations? Can hdf5 data be read? My Charades-STA options file is as follows. 
[opt.zip](https://github.com/jinhyunj/EaTR/files/12544919/opt.zip)

**Comment by wjun0830** [comment]
Yes right, but since self.feat_anet is not defined in the implementation, I defined with hd5 file by myself.
And by the way, is 0.33333 right for the clip length? Isn't it should be 0.5 when using I3d?

**Comment by jinhyunj** [comment]
Sorry for the late reply and confusion on the code. The data uploading code for the Charades-STA and ANet Caption datasets is the old version. I will update clear version of the code and config file as soon as possible.

**Comment by akzeycgdn265** [comment]
> Are you using start_end_dataset.py provided by EaTR for data reading operations? Can hdf5 data be read? My Charades-STA options file is as follows. [opt.zip](https://github.com/jinhyunj/EaTR/files/12544919/opt.zip)

Hello! Did you successfully run the code on the charades dataset? Could you tell me your wechat or email address? I want to ask you something about the code running!

**Comment by RobertLuo1** [comment]
Sorry for bothering, I wonder why the ActivityNet Caption Link can not be donwloaded and where are the txt_feature?

**Comment by 1343546245** [comment]
> Sorry for the late reply and confusion on the code. The data uploading code for the Charades-STA and ANet Caption datasets is the old version. I will update clear version of the code and config file as soon as possible.

Sorry for bothering, I wonder if you could give the config file of charades-STA?
